oauth2_client/tests/test_oauth2client_app_cmd.py

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
You may not need _app_ in these filenames - it reads well without it too.

Reply:
you mean only test files or the actual commands too? Please bear in mind that renaming files that contain commands will imply renaming the commands themselves too. Command rename would look like that:

oauth2client_app -> oauth2client
oauth2provider_app -> oauth2provider

I don't have enough experience with django naming conventions, I would be really grateful if you can make the call @auraz

Reply:
Only test, sorry for the confusion.

Reply:
Approved anyway as it is very minor.